p22/benches/mod.rs

```diff
@@ -0,0 +1,44 @@
+#![feature(test)]
+extern crate test;
+
+use p22::calc::celsius2farenheit;
+use p22::calc::farenheit2celsius;
+use p22::calc::fibonacci_loop;
+use p22::calc::fibonacci_rec;
+use test::{black_box, Bencher};
+
+#[bench]
+fn bench_celsius2farenheit(b: &mut Bencher) {
```

Comment:
Benchmarks were needed only for the Fibonacci functions, but it's fine to have these as well.